Buildings/Controls/OBC/FDE/DOAS/CoolingCoil.mo

```diff
@@ -0,0 +1,206 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block CoolingCoil "This block commands the cooling coil."
+
+
+  parameter Real erwDPadj(
+  final unit = "K",
+  final quantity = "TemperatureDifference") = 5
+  "Value subtracted from ERW supply air dewpoint.";
+
+  parameter CDL.Types.SimpleController controllerTypeDeh=Buildings.Controls.OBC.CDL.Types.SimpleController.PI
+    "Type of controller";
```

Comment:
Change descriptive comment to "Type of controller for dehumidification mode PI block". Similar comment for k and Ti

Reply:
When propagating parameters for PI block, please propagate Td parameter. You may have to temporarily set the type of controller parameter to PID to enable Td for propagation. You can change controller type back to the propagated parameter after that